deploy/argo/manifest/custom-dc-test.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: custom-dc-test
+  namespace: website
+spec:
+  project: default
+
+  syncPolicy:
+    automated:
+      # Delete resources that are no longer in the helm chart
+      prune: true
+      # When cluster is different from git, apply Git change to the cluster.
+      selfHeal: true
+
+
+  source:
+    repoURL: https://github.com/Fructokinase/website
+    targetRevision: argocd
+    path: deploy/helm_charts/dc_website
+
+    helm:
+      releaseName: dc-website
+
+      values : |
+        namespace:
+          name: website
+
+        website:
+          githash: "1421f9a"
```

Comment:
since githash is based on github tag and is always changing, do you need to check it in here? (It may give the confusion that this version is used)